_layouts/page.html

```diff
@@ -4,6 +4,13 @@
 
 <div class="container pb-5 mb-5">
     <div class="row align-items-center">
+      <!-- Outdated translation alert tool tag-->
+      <div class="col-md-12 order-md-1 text-md-left">
+        {% if page.outdated_translation %}
+          <p class="text-muted">⚠ This page contains some outdated translations. Help us improve <a href="https://www.transifex.com/bisq/bisq-website/">here</a>!</p>
```

Comment:
Don't make links like this:
<a href="/foo">Click here</a> for more information</a>
Make links like this:
<a href="/foo">More information</a>